### DISCUSSION TITLE
Add support for Adafruit 0.56" 4-Digit 7-Segment Display w/I2C Backpack

### DISCUSSION THREAD

**Discussion by alehuo** [MobiFlight/MobiFlight-Connector#1637] [Ideas]
**Is your feature request related to a problem? Please describe.**

This feature request is not related to any problem with MobiFlight connector.

**Describe the solution you'd like**

I happen to own a few Adafruit 0.56" 4-Digit 7-Segment Displays (w/I2C backpacks). They plug straight into Arduino boards and multiple of these can be daisy chained and operated via a single I2C connector: https://www.adafruit.com/product/1002 (These come in multiple color options).

**Describe alternatives you've considered**

An alternative would of course be, to use a compatible 7-segment LED, which has been tested to work with MobiFlight. As I own multiple of these expensive Adafruit 7-segment displays, I thought why not request support for it.

**Additional context**

Documentation: https://learn.adafruit.com/adafruit-led-backpack/0-dot-56-seven-segment-backpack

Note, that there are two different versions - the older one with traditional connectors and a newer one utilising StemmaQT. It would be nice to support both of these revisions.

**Comment by DocMoebiuz** [comment]
We have Max7219 and TM1637 - this is more exotic.  This could be implemented as community device. I will convert this into a discussion.
